validator/delphi_validator/utils.py

```diff
@@ -8,10 +8,11 @@
 GEO_REGEX_DICT = {
     'county': r'^\d{5}$',
     'hrr': r'^\d{1,3}$',
+    'hhs': r'^\d{1,2}$',
```

Comment:
why is there both regex and a static file?

Reply:
Not sure on the original design but it appears we are checking bad formatting vs. bad values but I agree it seems like overkill.

Reply:
got it. does it matter if the column is read in as int vs string vs zero padded string (e.g. 1, '1', '01')?

Reply:
geo ids are read as strings so 1 vs. '1' is not relevant.  The formatting will pass for '01' but the value will fail; we could change that to be a formatting failure (require the first digit to be non-zero).

Reply:
I have no real preference for how this is done.  It would be nice to only have to do value comparison and ignore formatting but it appears that we are guarding against geo ids that have encoded as ints or floats rather than strings.  It is a plausible design to trigger a failure if the geo ids do not match the strings due to a numeric conversion rather than try to reconstruct to the underlying string value.  @nmdefries might have some insight into the latitude we have here to be strict/lenient

Reply:
makes sense. The geomapper doesnt add leading 0's so i think this is fine for now.

Reply:
>  it appears that we are guarding against geo ids that have encoded as ints or floats

This is a nice-to-have. We want geo ids to be strings as the gold standard; ints and floats are tech debt but don't cause any actual problems besides inconsistent formatting. Value comparison is certainly more important.